### PR TITLE
picard: update to 2.10

### PR DESCRIPTION
**Pull request by rookiejet**
#### Testing the changes
- I tested the changes in this PR: **YES**

Fixes support for Python 3.12.

### CODE REVIEW

File: srcpkgs/picard/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'picard'
 pkgname=picard
-version=2.9.1
-revision=2
+version=2.10.0rc1
```
Comment (meator) [inline srcpkgs/picard/template:3]: The `2.10` version has been released today. The newer version should be preferred I believe.

**Review by meator — CHANGES_REQUESTED** [review]
Hi. I have run into the same issue with picard. I was preparing to update the template but then I found you already did it so thanks for that!

I have tested this PR myself and it works all right. The current version (`picard-2.9.1_2`) isn't functional due to the Python update so this PR should preferably be merged soon.

**Comment by meator** [comment]
Also ping the maintainer: @the-maldridge

File: srcpkgs/picard/template (hunk 1 above)
Comment (rookiejet) [reply srcpkgs/picard/template]: I updated it accordingly.

**Comment by lemmi** [comment]
Quickly tested it by tagging a single album. Seems to be working.

**Pull request merged**